experimental/parser/testdata/parser/import/in_message.proto.stderr.txt

```diff
@@ -0,0 +1,45 @@
+error: unexpected import within message definition
+  --> testdata/parser/import/in_message.proto:20:5
+   | 
+19 | / message M {
+20 | |     import "foo.proto";
+   | |     ^^^^^^^^^^^^^^^^^^^ this import...
+...  | 
+25 | | }
+   | \_- ...cannot be declared within this message definition
+
+error: unexpected public import within message definition
+  --> testdata/parser/import/in_message.proto:21:5
+   | 
+19 | / message M {
+20 | |     import "foo.proto";
+21 | |     import public "foo.proto";
+   | |     ^^^^^^^^^^^^^^^^^^^^^^^^^^ this public import...
+...  | 
+25 | | }
+   | \_- ...cannot be declared within this message definition
+
+error: unexpected weak import within message definition
+  --> testdata/parser/import/in_message.proto:22:5
+   | 
+19 | / message M {
+...  | 
+22 | |     import weak "foo.proto";
+   | |     ^^^^^^^^^^^^^^^^^^^^^^^^ this weak import...
+23 | | 
```

Comment:
Why does this diagnostic have an extra line of context after the snippet? The two above do not, so perhaps it's a bug?

If it's a quirk/bug in the reporting, totally fine to fix that in a separate PR. But, if it's an issue in the way the diagnostic is constructed in this commit, then let's try to fix in this branch.

Reply:
it's a quirk with the renderer, but it was easy enough to find and fix.